src/WebJobs.Extensions.DurableTask/ContextImplementations/DurableClient.cs

```diff
@@ -405,6 +406,52 @@ async Task IDurableOrchestrationClient.TerminateAsync(string instanceId, string
             }
         }
 
+        async Task IDurableOrchestrationClient.SuspendAsync(string instanceId, string reason)
+        {
+            OrchestrationState state = await this.GetOrchestrationInstanceStateAsync(instanceId);
+            if (IsOrchestrationSuspendable(state))
+            {
+                // TODO: will suspend events target a particular execution ID?
+                state.OrchestrationInstance.ExecutionId = null;
```

Comment:
Yeah I think `ExecutionId` helps us identify orchestrators that have gone through a `ContinueAsNew`-cycle.
I'm unsure of the interaction between suspend and `ContinueAsNew`. We'll need to think about that.
FYI - @amdeel: do you think we'll need to do anything special here?

Reply:
No, executionId is not something we want to use for this feature.